modules/plugin/jdbc/jdbc-hana/src/test/java/org/geotools/data/hana/HanaVirtualTableOnlineTest.java

```diff
@@ -0,0 +1,71 @@
+package org.geotools.data.hana;
```

Comment:
missing copyright/license header

Reply:
eg.
```
/*
 *    GeoTools - The Open Source Java GIS Toolkit
 *    https://geotools.org
 *
 *    (C) 2024, Open Source Geospatial Foundation (OSGeo)
 *
 *    This library is free software; you can redistribute it and/or
 *    modify it under the terms of the GNU Lesser General Public
 *    License as published by the Free Software Foundation;
 *    version 2.1 of the License.
 *
 *    This library is distributed in the hope that it will be useful,
 *    but WITHOUT ANY WARRANTY; without even the implied warranty of
 *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 *    Lesser General Public License for more details.
 */
```

Reply:
Thanks, added.